README.md

```diff
@@ -1,24 +1,73 @@
+* AD approval required - the resolution of the issue requires the AD to review and approve the change.
+* rfced-ready - the RFC Editor can update the document and/or close the issue based on the issue discussion.
+
+Feel free to add new issues and PRs for new edits and questions. 
+
+The editor will close the issue when it has been addressed either by accepting a pull request or simply closing the issue if no changes are needed. 
+
+## Pull Requests
+If you would like to submit changes, please create a pull request and identify one or more issues that the pull request closes in the comments. 
+
+If the editor creates the PR, the editor will expect feedback from the reviewer before merging the PR. 
+
+
+## Viewing Markdown Outputs
+Output files may be viewed here:
+* [https://www.rfc-editor.org/authors/rfc9345.txt](https://www.rfc-editor.org/authors/rfc9345.txt)
+
+Once the content of the .md file is stable and an XML file has been created, the following files will also be provided:
+* [https://www.rfc-editor.org/authors/rfc9345.html](https://www.rfc-editor.org/authors/rfc9345.html)
+* [https://www.rfc-editor.org/authors/rfc9345.pdf](https://www.rfc-editor.org/authors/rfc9345.pdf)
+* [https://www.rfc-editor.org/authors/rfc9345.xml](https://www.rfc-editor.org/authors/rfc9345.xml)
+   
+To create your own output files, use the [tools webservice](https://author-tools.ietf.org/) and select one of the output options.
+
+## Viewing Diffs
+Diffs via GitHub:
+* [All editing updates made before AUTH48](https://github.com/rfc-editor/rfc9345-AUTH48/compare/fe12bde..52bb939)
+
+More information on using GitHub to view diffs can be found here: [Comparing commits](https://docs.github.com/en/github/committing-changes-to-your-project/viewing-and-comparing-commits/comparing-commits).
+
+The following diff files are available via www.rfc-editor.org:
+* [https://www.rfc-editor.org/authors/rfc9345-diff.html](https://www.rfc-editor.org/authors/rfc9345-diff.html) (all changes since the document entered the RFC Editor Queue)
+* [https://www.rfc-editor.org/authors/rfc9345-rfcdiff.html](https://www.rfc-editor.org/authors/rfc9345-rfcdiff.html) (side-by-side view of all changes since the document entered the RFC Editor Queue)
+
+## Approving the Document 
+This document is being edited in Markdown.  Once the content is stable, the Markdown will be converted to XML and formatted as an RFC.  Your final approval means you approve both the content and format.
+
+To approve your RFC for publication, **please reply to the AUTH48 email** stating that you approve this RFC for publication.  Please use **REPLY ALL**, as all the parties CC’ed on the message need to see your approval.
+
+The details of the AUTH48 status of the document are here: [https://www.rfc-editor.org/auth48/rfc9345](https://www.rfc-editor.org/auth48/rfc9345)
+
+## About This Repo
+Note that this repo is available only during AUTH48. Before the RFC publication announcement is sent, this repo will be archived. 
```

Comment:
Since we're doing this in the WG repo, this paragraph is inaccurate.  We will keep this around in case of a -bis.